types/src/v0/impls/solver.rs

```diff
@@ -0,0 +1,71 @@
+use committable::{Commitment, Committable};
+use hotshot::types::SignatureKey;
+
+use super::v0_3::{RollupRegistrationBody, RollupUpdatebody};
+
+impl Committable for RollupRegistrationBody {
+    fn tag() -> String {
+        "ROLLUP_REG".to_string()
+    }
+
+    fn commit(&self) -> Commitment<Self> {
+        let active: [u8; 1] = if self.active { [1] } else { [0] };
```

Comment:
```suggestion
        let active: [u8; 1] = [u8::from(self.active)];
```

Reply:
done in 5e5c80536f400ed1adb96c7a01c23792c3dbb6b4